app/views/main.hbs

```diff
@@ -44,8 +48,20 @@
 
     {{! Web Monetization, if a payment pointer is provider (experimental) }}
     {{#config.monetization}}
-    <meta name="monetization" content="{{.}}">
+      <meta name="monetization" content="{{WEB_MONETIZATION_PAYMENT_POINTER}}">
     {{/config.monetization}}
+    <script>
+      if (document.monetization) {
+        document.monetizationExtensionInstalled = true
+      } else {
+        document.monetization = document.createElement('div')
+        document.monetization.state = 'stopped'
+      }
+    </script>
+    <script src="https://cdn.coil.com/coil-oauth-wm.v7.beta.js" defer>
```

Comment:
Since this is a new script, you'll need to add the domain to the list of allowed domains for CSP, located here: https://github.com/streetmix/streetmix/blob/main/app.js#L86

Reply:
thanks for this! I might need to pick your brain about the rest of this, as I'm getting some other errors with the scripts embedded inline as well